Makefile

```diff
@@ -1,8 +1,25 @@
-init:
+.PHONY: install-tox
+install-tox: ## Install tox into local python environment.
+	pip install tox
```

Comment:
init is for `make test` while `install-tox` is for `make tox`. It might make sense to add install-tox to init. I didn't that since tox might already be installed as a native os package and I wanted to let the dev choose what tox to use.

I kept it seperate from init since that is mainly responsible for calling setup.py and tox can't be installed through setup.py since it runs against multiple venvs.

Reply:
Fair enough.